lib/qa/authorities/linked_data/config/config_merge.rb

```diff
@@ -0,0 +1,76 @@
+module Qa::Authorities
+  module LinkedData
+    module ConfigMerge
```

Comment:
Add module documentation. What does this do? When do you use it? How? And a brief example would be 👍